bazel-build.el

```diff
@@ -24,6 +24,17 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-build nil
+  "Package for building and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-program "bazel"
+  "Name of the 'bazel' program for 'bazel-build' and 'bazel-test'."
+  :type 'string
+  :group 'bazel-build
+  :link '(url-link "https://bazel.build/"))
```

Comment:
Since this is the kind of thing that is often project local (i.e., in .dir-locals.el), it'd be useful to add a `:safe` field.

From the Emacs info page:
```
   When defining a user option using ‘defcustom’, you can set its
‘safe-local-variable’ property by adding the arguments ‘:safe FUNCTION’
to ‘defcustom’ (*note Variable Definitions::).  However, a safety
predicate defined using ‘:safe’ will only be known once the package that
contains the ‘defcustom’ is loaded, which is often too late.  As an
alternative, you can use the autoload cookie (*note Autoload::) to
assign the option its safety predicate, like this:

     ;;;###autoload (put 'VAR 'safe-local-variable 'PRED)

The safe value definitions specified with ‘autoload’ are copied into the
package’s autoloads file (‘loaddefs.el’ for most packages bundled with
Emacs), and are known to Emacs since the beginning of a session.
```